client/src/components/utils/LoginForm/index.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+import { BrowserRouter as Router, Link } from "react-router-dom";
+import { Form, Icon, Input, Checkbox } from "antd";
+import PropTypes from "prop-types";
+
+import Button from "../Button";
+import "./style.css";
+
+class LoginForm extends Component {
+  handleSubmit = e => {
+    e.preventDefault();
+    this.props.form.validateFields((err, values) => {
+      if (!err)
+        fetch("/login", { method: "POST", body: JSON.stringify(values) });
+    });
+  };
+
+  render() {
+    const { getFieldDecorator } = this.props.form;
+    return (
+      <Form onSubmit={this.handleSubmit} className="login-form">
+        <Form.Item label="Email Address">
+          {getFieldDecorator("username", {
+            rules: [
+              { required: true, message: "Email address is required!" },
+              { type: "email", message: "Please enter a valid email address" }
+            ]
+          })(
+            <Input
+              prefix={<Icon type="user" style={{ color: "rgba(0,0,0,.25)" }} />}
+              placeholder="example@example.com"
+              className="login-form--input"
+            />
+          )}
+        </Form.Item>
+
+        <Form.Item label="Password">
+          {getFieldDecorator("password", {
+            rules: [{ required: true, message: "Please enter your Password!" }]
+          })(
+            <Input
+              prefix={<Icon type="lock" style={{ color: "rgba(0,0,0,.25)" }} />}
+              type="password"
+              placeholder="Password"
+              className="login-form--input"
+            />
+          )}
+        </Form.Item>
+
+        <Form.Item>
+          {getFieldDecorator("rememberMe", {
+            valuePropName: "checked",
+            initialValue: false
+          })(<Checkbox>Remember me</Checkbox>)}
+        </Form.Item>
+
+        <div className="login-form--submit-div">
+          <Button type="submit">Log in</Button>
+          <Router>
+            <span className="form--create-acc">
+              Don't have an account?
+              <Link to="/signup" className="login-form--signup-link">
+                Sign up
+              </Link>
+              now
+            </span>
+          </Router>
+        </div>
+      </Form>
+    );
+  }
+}
+
+LoginForm.propTypes = {
```

Comment:
you don't need to check that
because you will get that from ``` antd ``` 

and  your code wrong btw that should be : 
 ```suggestion
LoginForm.propTypes = {
form: PropTypes.object.isRequired
}
```